developer-workflow/stdlib.rst

```diff
@@ -137,3 +138,17 @@ once the authors of the module sign
 :ref:`contributor agreements <contributor_agreement>`.
 
 .. _issue tracker: https://github.com/python/cpython/issues
+
+Adding a new env var
+--------------------
+
+From Python 3.13 onwards, name environment variables with underscores for
+readability and accessibility.
+
+For example, use ``PYTHON_CPU_COUNT`` instead of ``PYTHONCPUCOUNT``.
```

Comment:
```suggestion
For example, use ``PYTHON_CPU_COUNT`` instead of ``PYTHONCPUCOUNT``.
```
Not sure this deserves a separate paragraph.

Reply:
I think it's clearer like this. If we expand the section, we can adjust as needed.